docs/source/using/run-against-an-existing-machine.adoc

```diff
@@ -84,9 +84,16 @@ Use the following command on the host to run {project} against a remote machine:
 
 ----
 $ minishift start --vm-driver generic --remote-ipaddress <remote_IP_address> --remote-ssh-user <username> --remote-ssh-key <private_ssh_key>
+$ minishift addon apply htpasswd-identity-provider
 ----
 
 [NOTE]
 ====
 The value of the `--remote-ssh-key` flag must be the location of a private SSH key on the host machine.
 ====
+
+[NOTE]
+====
+Make sure you apply `htpasswd-identity-provider` add-on after minishift start, it will allow you to use htpasswd as identity provider.
```

Comment:
`as the identity provider`.

Reply:
To change the default username/password, you must apply `htpasswd-identity-provider` add-on with the new username/password after minishift start.

Reply:
cc @robin-owen please take a look and suggest.

Reply:
I agree with @LalatenduMohanty's suggestion with one addition -- I'd be more explicit about `minishift start`:

```adoc
To change the default username and password, you must apply the `htpasswd-identity-provider` add-on with the desired username and password after running the `minishift start` command.
```

If "and" is not technically accurate, "or" can be used instead. I'd avoid using "and/or".